src/borg/testsuite/archiver.py

```diff
@@ -4562,6 +4565,15 @@ def do_asserts(output, can_compare_ids, content_only=False):
             change = '0 B' if can_compare_ids else '{:<19}'.format('modified')
             self.assert_line_exists(lines, f"{change}.*input/empty")
 
+            # Do not show a 0 byte change for a file whose contents weren't
```

Comment:
Not a native speaker: is "whose" correct here or would that only be used for persons?

Reply:
Interesting question. As a native speaker, I never thought of it.

According to Wiki, it's [valid](https://en.wikipedia.org/wiki/Inanimate_whose), but ppl don't like it. Two alternate constructions:

* Do not show a 0 byte change for a file of which the contents weren't...
* Do not show a 0 byte change for a file the contents of which weren't...

(I prefer inanimate whose, but if forced to choose between the above two, I choose the former)